### DISCUSSION TITLE
Content persisting over a browser refresh ?

### DISCUSSION THREAD

**Discussion by casesolved-co-uk** [GrapesJS/grapesjs#5685] [Q&A]
I'm a newb so probably doing something obvious wrong, but I am trying to save/load templates to a database, but can't seem to stop it saving and loading the old content in the browser.

I have `autosave` and `autoload` set `false`. And I'm `editor.destroy`ing any old editors before creating a new one. Should I look into loading the template into an existing editor instead perhaps?

I'm not trying to use the builtin Storage engine; I'm using getJs, getHtml, etc to save and inserting the loaded content into the webpage before grapesjs.init-ing.

**Comment by artf** [comment]
Did you check already the [Storage](https://grapesjs.com/docs/modules/Storage.html) documentation?

> I'm not trying to use the builtin Storage engine; I'm using getJs, getHtml, etc to save and inserting the loaded content into the webpage before grapesjs.init-ing.

You should provide a reproducible demo as it might be something wrong with your custom code.

**Reply by casesolved-co-uk** [reply]
Thanks, yes my fault, `storageManager: false,` did the trick, thanks.
